pack/side/mtt_encounter.json

```diff
@@ -1,4 +1,258 @@
+        "back_name": "Imprisoned in Tindalos",
+        "back_text": "Each remaining investigator in all eras is defeated and suffers 1 mental trauma.\nIf the Edwin Bennet asset is in play in any era, proceed to <b>(→R2)</b>.\nIf the Edwin Bennet enemy is in play in any era, proceed to <b>(→R3)</b>.\nOtherwise, proceed to <b>(→R4)</b>.",
+        "code": "87003",
+        "doom": 9,
+        "double_sided": true,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 3,
+        "faction_code": "mythos",
+        "flavor": "The more you travel through Tindalos, the more likely its minions are to take notice and pursue you. If you don't complete your mission in time, you might become permanent guests of Tindalos.",
+        "illustrator": "Adam Doyle",
+        "name": "Time Marches On",
+        "pack_code": "mtt",
+        "position": 3,
+        "quantity": 1,
+        "stage": 2,
+        "text":"<b>Forced</b> - When a [[Scientist]] asset would be defeated: It is abducted instead <i>(set it aside, out of play)</i>.",
+        "type_code": "agenda"
+    },
+    {
+        "back_flavor": "You breathe a sigh of relief as you watch Thomas and Mary receive their well-earned award. The timeline appears to have been stitched back together, and things are as they should be. They thank you profusely, forever grateful for your integral contributions to their legacy. You wonder with cautious hope whether Edwin's machinations are truly over. The forces of Tindalos may never be defanged. You can only remain vigilant and hope that your meddling has caused no further paradox.",
+        "back_name": "Time Mends",
+        "back_text": "<b>(→R1)</b>",
+        "clues": null,
+        "code": "87004",
+        "double_sided": true,
+        "encounter_code": "machinations_through_time",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "flavor": "Nervously, you step through a portal and enter the realm of angular time and infinite depths.",
+        "illustrator": "Borja Pindado",
+        "name": "Walking Through Time",
+        "pack_code": "mtt",
+        "position": 4,
+        "quantity": 1,
+        "stage": 1,
+        "text": "Abilities on [[Scientist]] assets can be triggered by any investigator at that asset's location.\n[reaction] When you move to a location, if one or more [[Scientist]] assets are at your location: Move any number of them with you.\n<b>Objective</b> - Restore the timeline by completing the objective on each story card. If Thomas Corrigan and Mary Zielinski are at non-Tindalos locations in all 3 eras, and each era has no story cards in play, advance.",
+        "type_code": "act"
+    },
+    {
+        "clues": 0,
+        "code": "87005a",
```

Comment:
Some cases where there's an `a` and `b` printed on the card reflect this in the id (`87005a`, `87036a`, `87037a`), most don't (`87006`, `87015`, `87024`, `87033`, `87034`, `87035`). Is this intentional?